src/transport/WsOutboundTransporter.ts

```diff
@@ -0,0 +1,98 @@
+import { OutboundTransporter } from './OutboundTransporter'
+import { Agent } from '../agent/Agent'
+import { WebSocketTransport } from '../agent/TransportService'
+import { Logger } from '../logger'
+import { ConnectionRecord } from '../modules/connections'
+import { OutboundPackage } from '../types'
+import { Symbols } from '../symbols'
+import { WebSocket } from '../utils/ws'
+
+export class WsOutboundTransporter implements OutboundTransporter {
+  private transportTable: Map<string, WebSocket> = new Map<string, WebSocket>()
+  private agent: Agent
+  private logger: Logger
+
+  public supportedSchemes = ['ws', 'wss']
+
+  public constructor(agent: Agent) {
+    this.agent = agent
+    this.logger = agent.injectionContainer.resolve(Symbols.Logger)
+  }
+  public async start(): Promise<void> {
+    // Nothing required to start WS
+  }
+
+  public async stop() {
+    this.transportTable.forEach((socket) => {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
Why do we have `@ts-ignore` here? Is it because `WebSocket` from `ws` interface differs from ES standard and doesn't have `removeAllListeners`?

Reply:
Did some research. React Native / Browser websocket does not have remove all I believe. Updated